src/app/features/home/custom-camera/custom-camera.page.ts

```diff
@@ -154,16 +154,13 @@ export class CustomCameraPage implements OnInit, OnDestroy {
   }
 
   // eslint-disable-next-line class-methods-use-this
-  private scaleUpGalleryButton() {
-    // TODO: use @ViewChild elementRef
-    const element = document.getElementById('gallery-icon');
-    element?.classList.add('scaled-mat-icon');
-  }
-
-  // eslint-disable-next-line class-methods-use-this
-  private scaleDownGalleryButton() {
-    // TODO: use @ViewChild elementRef
-    const element = document.getElementById('gallery-icon');
-    element?.classList.remove('scaled-mat-icon');
+  private flashCameraScreen() {
+    const element = document.getElementById('camera-flash-placeholder');
+    element?.classList.add('flash-camera-animation');
+    setTimeout(
+      () => element?.classList.remove('flash-camera-animation'),
+      // eslint-disable-next-line @typescript-eslint/no-magic-numbers
```

Comment:
I think this lint error could be resolved by declaring something like `const flashCameraTimeout = 1000;`

Reply:
@shc261392, here is the commit [fix(custom-camera): extract magic number to meaninful var](https://github.com/numbersprotocol/capture-lite/pull/1745/commits/206c2d149efd7699326a1c7a4671522bf1913801)